src/main/java/com/ghilly/controller/CountryController.java

```diff
@@ -11,35 +11,40 @@
 @RequestMapping("/countries")
 public class CountryController {
 
+    CountryServiceRest service;
 
-    private final CountryService countryService;
-
-    public CountryController(CountryService countryService) {
-        this.countryService = countryService;
+    public CountryController(CountryServiceRest service){
+        this.service = service;
     }
 
+
     @PostMapping("/")
-    public void create(@RequestBody String name) {
-        System.out.println(name);
+    public void create(@RequestBody String countryName) {
+        service.add(countryName);
+        System.out.println("create" + countryName);
     }
 
     @GetMapping("/")
     public List<String> getCountries() {
+        service.receiveList();
+        System.out.println("get countries");
         return new ArrayList<>();
     }
 
     @GetMapping("/{countryId}")
-    public String getCountry(@PathVariable int countryId) {
-        return "get" + countryId;
+    public void getCountry(@PathVariable int countryId) {
+        service.receiveCountry(countryId);
```

Comment:
Why do you use sout in some methods but not here?

Reply:
Done.